### PR TITLE
Dataloader pytest

### PR DESCRIPTION
**Pull request by HagerSherif**

### CODE REVIEW

File: test/testdata/batches/image_batches/batch0_.npy
Comment (Muhammad-Al-Barbary) [inline test/testdata/batches/image_batches/batch0_.npy:1]: Why are there two testing folders? move the testdata folder to tests and delete the other one.

File: test_dataloader.py
Comment (Muhammad-Al-Barbary) [inline test_dataloader.py:1]: Move this file to the tests folder.

File: test_dataloader.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    assert isinstance(val_dataloader, monai.data.DataLoader)
+
+    # correct number of batches
+    assert len(train_dataloader) == len(os.listdir(image_batches_dir))   
+
+    for i,train_batch in enumerate(train_dataloader):  
+        expected_image=np.load( os.path.join(image_batches_dir, f"batch{i}_.npy"))
+        expected_label=np.load(os.path.join(label_batches_dir, f"batch{i}_.npy"))
+
+        assert train_batch['image'].shape== expected_image.shape  #correct batch dimension (batch_size,1,length,width)
```
Comment (Muhammad-Al-Barbary) [inline test_dataloader.py:56]: use the enum class (Keys) instead of the hardcoded keys,

**Review by Muhammad-Al-Barbary — COMMENTED** [review]
I like this one, good job!
Please try to stick to one style convention, you can use a linting tool.
Also, be smart while choosing/designing the test data. Aim for general-purpose cases that are suitable for multiple tests, as it won't be efficient to add new test cases for every new test.

**Comment by HagerSherif** [comment]
> Aim for general-purpose cases that are suitable for multiple tests, as it won't be efficient to add new test cases for every new test.

@Muhammad-Al-Barbary what can be changed in the test data ?

**Comment by Muhammad-Al-Barbary** [comment]
> > Aim for general-purpose cases that are suitable for multiple tests, as it won't be efficient to add new test cases for every new test.
> 
> @Muhammad-Al-Barbary what can be changed in the test data ?
This will be more clear when merging the tests and their respective test cases. You will most probably notice that the number of cases can be reduced by omitting the redundant ones and by using the same cases for multiple tests. There isn't much to do right now, but keep it in mind.

**Review by helrewaidy — APPROVED** [review]
Good job ya Hagar! Please address Mohammeds' comments.

File: test/testdata/batches/image_batches/batch0_.npy
Comment (HagerSherif) [reply test/testdata/batches/image_batches/batch0_.npy:1]: done

File: test_dataloader.py
Comment (HagerSherif) [reply test_dataloader.py:1]: done

File: test_dataloader.py (hunk 1 above)
Comment (HagerSherif) [reply test_dataloader.py]: done

**Review by Muhammad-Al-Barbary — APPROVED** [review]

**Pull request merged**